src/simplify.rs

```diff
@@ -18,12 +18,12 @@ pub fn escape_message_footer_marks(text: &str) -> String {
 
 /// Remove standard (RFC 3676, §4.3) footer if it is found.
 #[allow(clippy::indexing_slicing)]
-fn remove_message_footer<'a>(lines: &'a [&str]) -> &'a [&'a str] {
+fn remove_message_footer<'a>(lines: &'a [&str]) -> (&'a [&'a str], bool) {
```

Comment:
Maybe return an `Option` here, and convert `None` to `(lines, false)` outside? Or document the return type in the doc comment at least.

Reply:
i'll refine the documentation.